package.json

```diff
@@ -13,6 +13,7 @@
     "core-js": "^2.6.5",
     "register-service-worker": "^1.6.2",
     "vue": "^2.6.10",
+    "vue-apollo": "^3.0.0-beta.11",
```

Comment:
Can we use the latest stable version instead of beta?

Reply:
Yes we can.should I change the `package.json`?